### PR TITLE
ci: added settings to the cra scan

### PR DESCRIPTION
**Pull request by AnaBil-k**
### Description

added settings to the car scan

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: cra-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: "v1"
+CRA_TARGETS:
+  - CRA_TARGET: "solutions/full-stack"
+    PROFILE_ID: "0e6e7b5a-817d-4344-ab6f-e5d7a9c49520" # SCC profile ID (currently set to the FSCloud 1.4.0 profile).
+    CRA_IGNORE_RULES_FILE: ""
```
Comment (ocofaigh) [inline cra-config.yaml:5]: set this to `cra-tf-validate-ignore-rules.json`

**Review by surajsbharadwaj — APPROVED** [review]

**Review by surajsbharadwaj — APPROVED** [review]

**Pull request merged**